### ISSUE TITLE
Invalid use of role="rowheader" in column headers

### ISSUE THREAD

**Issue by zps938** [tonytomov/jqGrid#628]
The rows in the column headers are created with the role rowheader which is invalid according to http://www.w3.org/TR/wai-aria/roles#rowheader. They should be defined with role="row". Currently the JAWS screen reader is incorrectly reading the column headers due to this incorrect role.
